doc/source/raster_class.md

```diff
@@ -0,0 +1,337 @@
+```
+
+## Crop
+
+Cropping a {class}`~geoutils.Raster` is done through the {func}`~geoutils.Raster.crop` function, which enforces new {attr}`~geoutils.Raster.bounds`.
+
+```{important}
+As with all geospatial handling methods, the {func}`~geoutils.Raster.crop` function can be passed only a {class}`~geoutils.Raster` or {class}`~geoutils.Vector`
+as argument.
+
+See {ref}`core-match-ref` for more details.
+```
+
+The {func}`~geoutils.Raster.crop` function can also be passed a {class}`list` or {class}`tuple` of bounds (`xmin`, `ymin`, `xmax`, `ymax`). By default,
+{func}`~geoutils.Raster.crop` is done in-place.
+For more details, see the {ref}`specific section and function descriptions in the API<api-geo-handle>`.
+
+```{code-cell} ipython3
+# Crop raster to smaller bounds
+raster_crop = raster.crop(crop_geom=(0.3, 0.3, 1, 1), inplace=False)
+raster_crop
+```
+
+## Polygonize
+
+Polygonizing a {class}`~geoutils.Raster` is done through the {func}`~geoutils.Raster.polygonize` function, which converts target pixels into a multi-polygon
+{class}`~geoutils.Vector`.
+
+```{note}
+For a {class}`~geoutils.Mask`, {func}`~geoutils.Raster.polygonize` implicitly targets `True` values and thus does not require target pixels. See
+{ref}`mask-class-poly-overloaded`.
+```
+
+```{code-cell} ipython3
+# Polygonize all values lower than 100
+vector_lt_100 = (raster < 100).polygonize()
+vector_lt_100
+```
+
+## Proximity
+
+Computing proximity from a {class}`~geoutils.Raster` is done through by the {func}`~geoutils.Raster.proximity` function, which computes the closest distance
+to any target pixels in the {class}`~geoutils.Raster`.
+
+```{note}
+For a {class}`~geoutils.Mask`, {func}`~geoutils.Raster.proximity` implicitly targets `True` values and thus does not require target pixels. See
+{ref}`mask-class-prox-overloaded`.
+```
+
+```{code-cell} ipython3
+# Compute proximity from mask for all values lower than 100
+proximity_lt_100 = (raster < 100).proximity()
+proximity_lt_100
+```
+
+Optionally, instead of target pixel values, a {class}`~geoutils.Vector` can be passed to compute the proximity from the geometry.
+
+```{code-cell} ipython3
+# Compute proximity from mask for all values lower than 100
+proximity_lt_100_from_vector = raster.proximity(vector=vector_lt_100)
+proximity_lt_100_from_vector
+```
+
+## Interpolate or extract to point
+
+Interpolating or extracting {class}`~geoutils.Raster` values at specific points can be done through:
+- the {func}`~geoutils.Raster.value_at_coords` function, who extracts the single closest pixel or a surrounding window for each coordinate, on which
+  can be applied reducing any function ({func}`numpy.ma.mean` by default), or
+- the {func}`~geoutils.Raster.interp_points` function, who interpolates the {class}`~geoutils.Raster`'s regular grid to each coordinate using a
```

Comment:
-> "that interpolates the Raster's gridded values at any given coordinate using the set resampling algorithm (by default linear)."

Reply:
Done